docs/manual/cli.qmd

```diff
@@ -0,0 +1,151 @@
+```{python}
+#| echo: false
+from odm_sharing.tools.share import OutFmt, share
+
+from common import DATA, SCHEMA, load_csv_md, print_file
+```
+
+# CLI {#sec-cli}
+
+## Reference
+
+```bash
+odm-share [OPTION]... SCHEMA INPUT
+```
+
+Arguments:
+
+- SCHEMA
+
+  sharing schema file path
+
+- INPUT
+
+  spreadsheet file path or [SQLAlchemy database URL](https://docs.sqlalchemy.org/en/20/core/engines.html#database-urls)
```

Comment:
We don't currently support anything other than spreadsheets right?

Reply:
it should also work to connect directly to databases

Reply:
I ran the following cli command, 

```
odm-share ./schema.csv postgresql+psycopg2://yulricsequeira@localhost:5432/odm
```

and I'm getting the following error,

```
unable to infer output format from input file
ERROR:root:unable to infer output format from input file
```

Reply:
good catch. that error should be fixed now. I can add an issue for testing db use properly as well.

Reply:
#21

Reply:
Should we remove all the db text or add text saying that the db code has not been tested yet?

Reply:
I think it's fine to keep it as is, since I'll make sure it works as soon as we finish these initial PRs. Is that ok for you?

Reply:
if it doesn't work it's just because of bugs.